test/NuGet.Core.Tests/NuGet.Build.Tasks.Console.Test/MSBuildStaticGraphRestoreTests.cs

```diff
@@ -101,6 +101,19 @@ public void GetOriginalTargetFrameworks_WhenTargetFramworksSpecified_HasCorrectT
             actual.Should().BeEquivalentTo(expected);
         }
 
+        [Theory]
+        [InlineData("net40;net45")]
+        [InlineData("net40;net45 ; netstandard2.0 ")]
+        public void GetOriginalTargetFramework_WhenTargetFramworkSpecified_HasCorrectTargetFramework(string targetFrameworks)
```

Comment:
```suggestion
        public void GetOriginalTargetFramework_WhenMultipleTargetFramworkSpecified_ThrowRestoreCommandException(string targetFrameworks)
```

Might want to change the name of this test.